### PR TITLE
include non flatten attributes into event

### PR DESCRIPTION
**Pull request by AlvaroVega**
Issue: https://github.com/telefonicaid/perseo-fe/issues/377

original event:

iot-perseo-core             | time=2019-10-07T07:35:56.316Z | lvl=INFO | from=::ffff:172.17.0.26 | corr=1961aebe-e8d5-11e9-8f21-0242ac11001a; perseocep=15 | trans=9a0c81b4-0ef2-430a-b510-3dc04b4929a0 | srv=smartcity | subsrv=/ | op=doPost | comp=perseo-core | msg=incoming event:{"noticeId":"19630a70-e8d5-11e9-95a4-6976f0d93149","noticeTS":1570433756311,"id":"thing:disp1","type":"thing","isPattern":"false","subservice":"/","service":"smartcity","TimeInstant":"2019-10-07T07:30:20.00Z","TimeInstant__type":"DateTime","TimeInstant__ts":1570433420000,"TimeInstant__day":7,"TimeInstant__month":10,"TimeInstant__year":2019,"TimeInstant__hour":7,"TimeInstant__minute":30,"TimeInstant__second":20,"TimeInstant__millisecond":0,"TimeInstant__dayUTC":7,"TimeInstant__monthUTC":10,"TimeInstant__yearUTC":2019,"TimeInstant__hourUTC":7,"TimeInstant__minuteUTC":30,"TimeInstant__secondUTC":20,"TimeInstant__millisecondUTC":0,"att":null,"att__type":"Mitipo","location__type":"string","location__coordinates__0":23,"location__coordinates__1":28,"open":true,"open__type":"Boolean","temp":77,"temp__type":"Number","temperature":13,"temperature__type":"Number","attr__color":"azul","attr__forma":"cuadrado","attr__type":"JSON"}

event with stripped data:

iot-perseo-core             | time=2019-10-07T07:35:56.316Z | lvl=INFO | from=::ffff:172.17.0.26 | corr=1961aebe-e8d5-11e9-8f21-0242ac11001a; perseocep=15 | trans=9a0c81b4-0ef2-430a-b510-3dc04b4929a0 | srv=smartcity | subsrv=/ | op=doPost | comp=perseo-core | msg=incoming event:{"noticeId":"19630a70-e8d5-11e9-95a4-6976f0d93149","noticeTS":1570433756311,"id":"thing:disp1","type":"thing","isPattern":"false","subservice":"/","service":"smartcity","TimeInstant":"2019-10-07T07:30:20.00Z","TimeInstant__type":"DateTime","TimeInstant__ts":1570433420000,"TimeInstant__day":7,"TimeInstant__month":10,"TimeInstant__year":2019,"TimeInstant__hour":7,"TimeInstant__minute":30,"TimeInstant__second":20,"TimeInstant__millisecond":0,"TimeInstant__dayUTC":7,"TimeInstant__monthUTC":10,"TimeInstant__yearUTC":2019,"TimeInstant__hourUTC":7,"TimeInstant__minuteUTC":30,"TimeInstant__secondUTC":20,"TimeInstant__millisecondUTC":0,"att":null,"att__type":"Mitipo","location__type":"string","location__coordinates__0":23,"location__coordinates__1":28,"open":true,"open__type":"Boolean","temp":77,"temp__type":"Number","temperature":13,"temperature__type":"Number","attr__color":"azul","attr__forma":"cuadrado","attr__type":"JSON",**"stripped":
[{"name":"TimeInstant","type":"DateTime","value":"2019-10-07T07:52:20.00Z"},{"name":"att","type":"Mitipo","value":null},{"name":"attr","type":"JSON","value":{"color":"azul","forma":"cuadrado"}},{"name":"location","type":"string","value":{"type":"Point","coordinates":[23,28]}},{"name":"open","type":"Boolean","value":true},{"name":"temp","type":"Number","value":77},{"name":"temperature","type":"Number","value":13}]**}

### CODE REVIEW

**Comment by AlvaroVega** [comment]
attributes array is accesible in "stripped", but ¿makes sense? 
An array of attributes could not be useful to be used in rule...

**Comment by fgalan** [comment]
>  An array of attributes could not be useful to be used in rule...

It would be better to use a key-values object (NGSIv2 style)

**Comment by AlvaroVega** [comment]
> > An array of attributes could not be useful to be used in rule...
> 
> It would be better to use a key-values object (NGSIv2 style)

Yes it will be NGSIv2 style if notification is done using v2

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Include stripped (non flatten) data into event (#377)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Maybe documentation should be modified? Maybe the following sections are impacted by the new functionality:

https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md#metadata-and-object-values
https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md#location-fields
https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md#time-fields
https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md#json-and-array-fields

(Not sure if I'm right. Should be checked)

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE]: fixed in https://github.com/telefonicaid/perseo-fe/pull/393/commits/74cd861da01df386697e5fade1e3a14670a59677

**Pull request merged**